packages/odyssey-storybook/src/components/Field/Field.stories.tsx

```diff
@@ -28,11 +28,14 @@ export default {
   args: {
     label: "Destination",
     optionalLabel: "Optional",
```

Comment:
Having this as part of the default arguments here makes the code examples confusing for the Field docs, e.g.
```
<WithStyles(Field)
  hint="Your planetary destination."
  optionalLabel="Optional"
  required
/>
```

Consider removing this and including it only where it is applicable for all the stories below. We might need to update types as well to have the logic we want enforced in code, that is `optionalLabel` is only required to be provided when a Field is `required=false | undefined`.

Reply:
I've reworked the stories here to not utilize default args since `required` is the only default we expect (per the component).

Agreed on the internal logic around `optionalLabel` - should be required when the Field/Input isn't. Ticket here, so this doesn't get lost: https://oktainc.atlassian.net/browse/OKTA-453308